etc/profile-m-z/seahorse.profile

```diff
@@ -26,14 +24,13 @@ include disable-xdg.inc
 #mkdir ${HOME}/.ssh
 #whitelist ${HOME}/.gnupg
 #whitelist ${HOME}/.ssh
-whitelist /tmp/ssh-*
 whitelist /usr/share/gnupg
 whitelist /usr/share/gnupg2
 whitelist /usr/share/seahorse
 whitelist /usr/share/seahorse-nautilus
 whitelist ${RUNUSER}/gnupg
 whitelist ${RUNUSER}/keyring
-#include whitelist-common.inc
+include whitelist-common.inc
```

Comment:
What about the commented `whitelist ${HOME}` commands and the nearby comment?

> whitelisting in ${HOME} breaks file encryption feature of nautilus.

Reply:
Argh, leftover from my testing. Thanks!